README.md

```diff
@@ -70,18 +70,18 @@ Supported Compilers
+| Serial                                                                         | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark:     | :white_check_mark:      | :white_check_mark:                              | :white_check_mark:                               | :white_check_mark:                       | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark: | :white_check_mark:                                    | :white_check_mark:                        | :white_check_mark:                   |
+| OpenMP 2.0+ blocks                                                             | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark:     | :white_check_mark:      | :white_check_mark:                              | :white_check_mark:                               | :white_check_mark:                       | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark: | :x:                                                   | :white_check_mark:                        | :white_check_mark:                   |
+| OpenMP 2.0+ threads                                                            | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark:     | :white_check_mark:      | :white_check_mark:                              | :white_check_mark:                               | :white_check_mark:                       | :white_check_mark:                        | :white_check_mark:                   | :white_check_mark: | :x:                                                   | :white_check_mark:                        | :white_check_mark:                   |
+| OpenMP 5.0 (CPU)                                                               | :x:                                       | :x:                                       | :x:                                       | :x:                                  | :x:                    | :x:                     | :x:                                             | :x:                                              | :white_check_mark:                       | :white_check_mark:                        | -                                    | :x:                | :x:                                                   | :x:                                       | :x:                                  |
```

Comment:
`icpx` not supporting OpenMP 5 and TBB seems weird to me. Have you just not tested this or are you positive it doesn't work?

Reply:
I did not test it. I will create an issue and send a subsequent PR.

Reply:
It's here: #1705